### PR TITLE
Documentation cleanup

### PR DESCRIPTION
**Pull request by AaronFeickert**
## Description

Cleans up the front-facing documentation.

### Type of Change

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature/Enhancement (non-breaking change which adds functionality or enhances an existing one)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [x] Documentation update
-   [ ] Refactor

## Checklist

-   [x] I have performed a self-review of my code.
-   [x] I have commented my code where necessary.
-   [x] I have updated the documentation if needed.
-   [x] My changes do not introduce new warnings.
-   [ ] I have added tests that prove my changes are effective or that my feature works.
-   [x] New and existing tests pass with my changes.

## Related Issues

None.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
 ## Transaction Graph
 
-The following is the transaction graph that has been implemented in this repo.
+The following is the transaction graph that has been implemented in this repository:
 
 <figure>
-    <img src="./assets/poc-tx-graph.jpg" alt = "poc tx graph" />
-    <figcaption>The Transaction Graph for this PoC focussing on the non-optimistic path.</figcaption>
+    <img src="./assets/poc-tx-graph.jpg" alt = "Transaction graph" />
+    <figcaption>The transaction graph for this repository, focusing on the non-optimistic path.</figcaption>
```
Comment (Rajil1213) [inline README.md:19]:
```suggestion
    <figcaption>The transaction graph for this repository, focussing on the non-optimistic path.</figcaption>
```

**Review by Rajil1213 — CHANGES_REQUESTED** [review]
Minor typo, otherwise looks fine.

File: README.md (hunk 1 above)
Comment (AaronFeickert) [reply README.md:19]: Ah, I wasn't aware this was a (presumably non-American English) spelling!

Comment (Rajil1213) [reply README.md]: I wasn't aware that's the American spelling. 😅

Fine to leave as is then.

Comment (AaronFeickert) [reply README.md]: I have no preference. Apparently both spellings are generally considered valid. Already reverted.

**Review by Rajil1213 — DISMISSED** [review]

**Comment by storopoli** [comment]
I would add links to these resources:

```md
-   SP1 Toolchain and associated linkers.
-   Credentials to connect to SP1's infra.
-   Docker.
-   `sqlx-cli` to run migrations.
```

**Comment by Rajil1213** [comment]
Rebased this PR with `main` and also added links to the prerequisites as suggested by Jose.

**Comment by storopoli** [comment]
> Rebased this PR with `main` (for a CI fix) and also added links to the prerequisites as suggested by Jose.

Thank you.

**Pull request merged**